### PR TITLE
Fix Size Container

### PR DESCRIPTION
**Pull request by jaluowen**

### CODE REVIEW

File: lib/screens/selfReport/SelfReportScreen.dart [hunk 1]
```diff
@@ -403,7 +403,7 @@ class _SelfReportScreenState extends State<SelfReportScreen> {
                               physics: NeverScrollableScrollPhysics(),
                               gridDelegate:
                                   SliverGridDelegateWithMaxCrossAxisExtent(
-                                      childAspectRatio: 1.4,
+                                      childAspectRatio: 5.6 / 4,
```
Comment (anandarh) [inline lib/screens/selfReport/SelfReportScreen.dart:406]: @jaluowen try to make a custom delegate, so it doesn't use aspect ratio.

```

class SliverGridDelegateWithFixedCrossAxisAndHeight
    extends SliverGridDelegate {
    
  /// Creates a delegate that makes grid layouts with a fixed number 
  /// and height of tiles in the cross axis.
  ///
  /// The `crossAxisCount` must not be null. The `mainAxisSpacing` and
  /// `crossAxisSpacing` arguments must not be negative. The `crossAxisCount`
  /// and `height` arguments must be greater than zero.
  const SliverGridDelegateWithFixedCrossAxisCountAndFixedHeight({
    @required this.crossAxisCount,
    this.mainAxisSpacing = 0.0,
    this.crossAxisSpacing = 0.0,
    this.height = 56.0,
  })  : assert(crossAxisCount != null && crossAxisCount > 0),
        assert(mainAxisSpacing != null && mainAxisSpacing >= 0),
        assert(crossAxisSpacing != null && crossAxisSpacing >= 0),
        assert(height != null && height > 0);

  /// The number of children in the cross axis.
  final int crossAxisCount;

  /// The number of logical pixels between each child along the main axis.
  final double mainAxisSpacing;

  /// The number of logical pixels between each child along the cross axis.
  final double crossAxisSpacing;

  /// The height of the crossAxis.
  final double height;

  bool _debugAssertIsValid() {
    assert(crossAxisCount > 0);
    assert(mainAxisSpacing >= 0.0);
    assert(crossAxisSpacing >= 0.0);
    assert(height > 0.0);
    return true;
  }

  @override
  SliverGridLayout getLayout(SliverConstraints constraints) {
    assert(_debugAssertIsValid());
    final double usableCrossAxisExtent =
        constraints.crossAxisExtent - crossAxisSpacing * (crossAxisCount - 1);
    final double childCrossAxisExtent = usableCrossAxisExtent / crossAxisCount;
    final double childMainAxisExtent = height;
    return SliverGridRegularTileLayout(
      crossAxisCount: crossAxisCount,
      mainAxisStride: childMainAxisExtent + mainAxisSpacing,
      crossAxisStride: childCrossAxisExtent + crossAxisSpacing,
      childMainAxisExtent: childMainAxisExtent,
      childCrossAxisExtent: childCrossAxisExtent,
      reverseCrossAxis: axisDirectionIsReversed(constraints.crossAxisDirection),
    );
  }

  @override
  bool shouldRelayout(
      SliverGridDelegateWithFixedCrossAxisCountAndFixedHeight oldDelegate) {
    return oldDelegate.crossAxisCount != crossAxisCount ||
        oldDelegate.mainAxisSpacing != mainAxisSpacing ||
        oldDelegate.crossAxisSpacing != crossAxisSpacing ||
        oldDelegate.height != height;
  }
}

```

**Review by anandarh — CHANGES_REQUESTED** [review]

File: lib/screens/selfReport/SelfReportScreen.dart (hunk 1 above)
Comment (jaluowen) [reply lib/screens/selfReport/SelfReportScreen.dart]: Done mas bram

Comment (anandarh) [reply lib/screens/selfReport/SelfReportScreen.dart]: Is it safe?

Comment (jaluowen) [reply lib/screens/selfReport/SelfReportScreen.dart]: Need to try on another device, but on my device it's working well

**Review by anandarh — APPROVED** [review]

**Comment by anandarh** [comment]
@jaluowen Masih terpotong ternyata di ukuran device 5".

**Comment by jaluowen** [comment]
iya udah dibenerin lagi mas bram, coba skrg masih engga? @anandarh

**Comment by anandarh** [comment]
> iya udah dibenerin lagi mas bram, coba skrg masih engga? @anandarh

Oke wait

**Review by anandarh — APPROVED** [review]

**Review by gaganugraha33 — APPROVED** [review]

**Pull request merged**